packages/datetime/test/dateRangePickerTests.tsx

```diff
@@ -311,6 +492,37 @@ describe("<DateRangePicker>", () => {
             assert.isTrue(onDateRangePickerChangeSpy.calledOnce);
         });
 
+        it("onHoverChange fired with correct values when a day is clicked", () => {
+            const dateRange = [new Date(2015, Months.JANUARY, 1), new Date(2015, Months.JANUARY, 5)] as DateRange;
+            renderDateRangePicker({ initialMonth: new Date(2015, Months.JANUARY, 1) });
+            assert.isTrue(onDateRangePickerHoverChangeSpy.notCalled);
+            clickDay(1);
+            assert.isTrue(onDateRangePickerHoverChangeSpy.calledOnce);
+            assert.isTrue(DateUtils.areSameDay(dateRange[0], onDateRangePickerHoverChangeSpy.args[0][0][0]));
+            assert.isNull(onDateRangePickerHoverChangeSpy.args[0][0][1]);
```

Comment:
Good point. I copied this syntax from an existing test in this file, but `calledWith` seems semantically clearer to me in general.